identity/id.go

```diff
@@ -24,7 +25,7 @@ func (id ID) Bytes() []byte {
 
 // String returns a shortened version of the ID as a hex encoded string.
```

Comment:
update comment, plus should it have a more descriptive name like `ShortString`? I would expect `String` to give me the full id string representation.

Reply:
I think it should stay that way because that's the string representation we decided to go with. So when we print it, it should always be the shorthand version.